test_data/ontologyR2RV2/knoraApiOntologyWithValueObjects.ttl

```diff
@@ -2093,68 +2202,138 @@ knora-api:MovingImageFileValue
         rdfs:subClassOf         [ a                      owl:Restriction ;
                                   knora-api:isInherited  true ;
                                   owl:cardinality        1 ;
-                                  owl:onProperty         knora-api:userHasPermission
+                                  owl:onProperty         knora-api:hasPermissions
                                 ] ;
         rdfs:subClassOf         [ a                      owl:Restriction ;
                                   knora-api:isInherited  true ;
-                                  owl:maxCardinality     1 ;
-                                  owl:onProperty         knora-api:deleteDate
+                                  owl:cardinality        1 ;
+                                  owl:onProperty         knora-api:attachedToUser
+                                ] ;
+        rdfs:subClassOf         [ a                   owl:Restriction ;
+                                  owl:maxCardinality  1 ;
+                                  owl:onProperty      knora-api:documentFileValueHasPageCount
```

Comment:
As mentioned elsewhere, this property seems somewhat strange to me for a bundle representation.